src/evict/evict_lru.c

```diff
@@ -1967,7 +1970,8 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
          */
         give_up = !__wt_cache_aggressive(session) && !WT_IS_HS(btree->dhandle) &&
           pages_seen > min_pages &&
-          (pages_queued == 0 || (pages_seen / pages_queued) > (min_pages / target_pages));
+          (pages_queued + pages_urgent_queued == 0 ||
```

Comment:
This is not only a stat change. What's the intention behind this particular change?